app/models/calendar_event.rb

```diff
@@ -51,10 +51,10 @@ def self.new_project_end(project)
     project.end_date ? new.initialize_project_end(project) : nil
   end
 
-  def self.filtered_events(date_range: nil, project_filter: nil, project_scope: Project, step_scope: ProjectStep)
+  def self.filtered_events(date_range: nil, project_filter: nil, project_scope: Project, step_scope: ProjectStep, project_id: nil)
```

Comment:
Try to keep lines no longer than 100 characters long.

There's a few lines where this happens, so just wrap them, as much as is possible.

Usual syntax is below:

````
def some_method
    events = variable(really long params,
        more params)
````